packages/angular/cli/models/architect-command.ts

```diff
@@ -74,12 +75,13 @@ export abstract class ArchitectCommand<
       }
     }
 
-    if (targetProjectNames.length === 0) {
-      throw new Error(`No projects support the '${this.target}' target.`);
+    if (!options.help && (targetProjectNames.length === 0)) {
```

Comment:
Why does `!options.help` need to added to this condition and the one below it? If I call `ng deploy --help` this still looks like the error that shows up.

Reply:
Since we don't have a default dummy builder, the command is going to always throw when the user invokes `ng deploy --help`.